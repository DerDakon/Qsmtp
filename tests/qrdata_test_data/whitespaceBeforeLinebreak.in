Content-type: text/plain; charset="UTF-8"
Content-Transfer-Encoding: 8bit

To force recoding: €€

lines with whitespace and safe characters, should trigger the "add one more character" branch in recode_qp().

 a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a
a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a

The same with unsafe characters so this doesn't trigger falsely:

 = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = =
= = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = =

Ok, this time with UTF8 data:

 ä ö ü ä ö ü ä ö ü ä ö ü ä ö ü ä ö ü ä ö ü ä ö ü ä ö ü ä ö ü ä ö ü ä ö ü ä ö ü ä ö ü ä ö ü ä ö ü
ä ö ü ä ö ü ä ö ü ä ö ü ä ö ü ä ö ü ä ö ü ä ö ü ä ö ü ä ö ü ä ö ü ä ö ü ä ö ü ä ö ü ä ö ü ä ö ü
