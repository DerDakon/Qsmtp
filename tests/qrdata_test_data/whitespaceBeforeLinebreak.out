Content-type: text/plain; charset="UTF-8"
Content-Transfer-Encoding: quoted-printable (recoded by: Qremote @QSMTP_VERSION@ at foo.bar.example.com)

To force recoding: =E2=82=AC=E2=82=AC

lines with whitespace and safe characters, should trigger the "add one mo=
re character" branch in recode_qp().

 a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a=
 a a a a a a a a
a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a=
 a a a a a a a a a

The same with unsafe characters so this doesn't trigger falsely:

 =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D=20=
=3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D=
 =3D =3D =3D =3D =3D =3D =3D =3D =3D
=3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D=
 =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D =3D=20=
=3D =3D =3D =3D =3D =3D =3D =3D =3D =3D

Ok, this time with UTF8 data:

 =C3=A4 =C3=B6 =C3=BC =C3=A4 =C3=B6 =C3=BC =C3=A4 =C3=B6 =C3=BC =C3=A4 =C3=
=B6 =C3=BC =C3=A4 =C3=B6 =C3=BC =C3=A4 =C3=B6 =C3=BC =C3=A4 =C3=B6 =C3=BC=
 =C3=A4 =C3=B6 =C3=BC =C3=A4 =C3=B6 =C3=BC =C3=A4 =C3=B6 =C3=BC =C3=A4 =C3=
=B6 =C3=BC =C3=A4 =C3=B6 =C3=BC =C3=A4 =C3=B6 =C3=BC =C3=A4 =C3=B6 =C3=BC=
 =C3=A4 =C3=B6 =C3=BC =C3=A4 =C3=B6 =C3=BC
=C3=A4 =C3=B6 =C3=BC =C3=A4 =C3=B6 =C3=BC =C3=A4 =C3=B6 =C3=BC =C3=A4 =C3=
=B6 =C3=BC =C3=A4 =C3=B6 =C3=BC =C3=A4 =C3=B6 =C3=BC =C3=A4 =C3=B6 =C3=BC=
 =C3=A4 =C3=B6 =C3=BC =C3=A4 =C3=B6 =C3=BC =C3=A4 =C3=B6 =C3=BC =C3=A4 =C3=
=B6 =C3=BC =C3=A4 =C3=B6 =C3=BC =C3=A4 =C3=B6 =C3=BC =C3=A4 =C3=B6 =C3=BC=
 =C3=A4 =C3=B6 =C3=BC =C3=A4 =C3=B6 =C3=BC
.
